modular_citadel/code/modules/clothing/under/syndicate.dm

```diff
@@ -0,0 +1,8 @@
+/obj/item/clothing/under/syndicate/tacticool
+	name = "tacticool turtleneck"
+	desc = "Just looking at it makes you want to buy an SKS, go into the woods, and -operate-."
```

Comment:
You don't need to redefine all of this, only the has_sensor

Reply:
Okiedoke